### ISSUE TITLE
[Issue] No bars after latest update

### ISSUE THREAD

**Issue by Yasuhan** [end-4/dots-hyprland#388]
- **(For usage) I have read the [wiki](https://end-4.github.io/dots-hyprland-wiki/en/i-i/02usage)**
- **(For troubleshooting) I have read the [wiki](https://end-4.github.io/dots-hyprland-wiki/en/i-i/03troubleshooting)**
- Linux distro: 
Arch
---
## The issue
After the latest update to multi-monitor the bars just stopped working completely. If I go back to like 13ab541fcbf5a49097a149dfe66579690975c0a3 it works without issue. Here are the logs, of which there really isn't a lot. 

<details>
<summary>Click to expand</summary>

```plain
Gjs-Console-Message: 13:34:26.346: [LOG] Styles loaded

(com.github.Aylur.ags:3234): Gjs-WARNING **: 13:34:26.483: Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:
Bar@file:///home/AAA/.config/ags/modules/bar/main.js:37:26
forMonitors@file:///home/AAA/.config/ags/config.js:26:24
@file:///home/AAA/.config/ags/config.js:73:12
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34


(com.github.Aylur.ags:3234): Gjs-WARNING **: 13:34:26.483: Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:
Bar@file:///home/AAA/.config/ags/modules/bar/main.js:37:26
forMonitors@file:///home/AAA/.config/ags/config.js:26:24
@file:///home/AAA/.config/ags/config.js:73:12
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34


(com.github.Aylur.ags:3234): Gjs-WARNING **: 13:34:26.483: Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:
Bar@file:///home/AAA/.config/ags/modules/bar/main.js:37:26
forMonitors@file:///home/AAA/.config/ags/config.js:26:24
@file:///home/AAA/.config/ags/config.js:73:12
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34
```

</details>

**Comment by end-4** [comment]
can you update the config then try again and send the output? i just added proper error logging for the bar in ea514e9a85778e9f006e4545d1844951e8741ca5

**Comment by Yasuhan** [comment]
Okay, managed to fix it through that. For some reason not all of my config files got properly updated and thus there was a reference error that I was only able to see with the latest commit. 

```plain
ReferenceError: OptionalWindowTitleInstance is not defined
```

Managed to sort that out myself now.

Interestingly with 'forMonitorsAsync(Bar);', all 3 bars will appear on the same monitor - so I changed it back to the prior thing now.
